cmd/logs.go

```diff
@@ -14,16 +14,18 @@ var (
 		Usage:     "Get the logs of your applications",
 		Description: `Get the logs of your applications
    Example:
-     Get 100 lines:  'scalingo --app my-app logs -n 100'
-     Real-Time logs: 'scalingo --app my-app logs -f'`,
+     Get 100 lines:          'scalingo --app my-app logs -n 100'
+     Real-Time logs:         'scalingo --app my-app logs -f'
+     Get lines with filter:  'scalingo --app my-app logs -n 100 -F web-1|worker-2'`,
 		Flags: []cli.Flag{appFlag,
 			cli.IntFlag{Name: "lines, n", Value: 20, Usage: "Number of log lines to dump", EnvVar: ""},
 			cli.BoolFlag{Name: "follow, f", Usage: "Stream logs of app, (as \"tail -f\")", EnvVar: ""},
+			cli.StringFlag{Name: "filter, F", Usage: "Filter process that will be displayed", EnvVar: ""},
```

Comment:
"Filter process that will be displayer" -> "Filter container logs that will be displayed"